### ISSUE TITLE
Activities of folders and files are displayed twice on second click

### ISSUE THREAD

**Issue by jabouchleih** [nextcloud/server#11261]
### Steps to reproduce
1. Click on the folder or file icon
2. 'Activities' view appears 
3. Reclick on folder
4. Activities in 'Activities' are displayed twice 

### Expected behaviour
Activities are only displayed once. 

### Actual behaviour
Activities are only displayed twice. 

### Server configuration

**Operating system**: Debian 8 (using the Docker image)

**Web server:** Apache (Reverse Proxy)

**Database:** MySQL (using the Docker image)

**PHP version:** Unknown - using the Docker image

**Nextcloud version:** 14

**Updated from an older Nextcloud/ownCloud or fresh install:** Coming from 13.0.6

**Where did you install Nextcloud from:** Docker

**Signing status:**
<details>
<summary>Signing status</summary>

```
No errors have been found.
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
Enabled:
  - accessibility: 1.0.1
  - activity: 2.7.0
  - audioplayer: 2.3.2
  - bookmarks_fulltextsearch: 0.99.2
  - calendar: 1.6.2
  - cloud_federation_api: 0.0.1
  - comments: 1.4.0
  - contacts: 2.1.6
  - dav: 1.6.0
  - drop_account: 0.0.11
  - federatedfilesharing: 1.4.0
  - federation: 1.4.0
  - files: 1.9.0
  - files_fulltextsearch: 0.99.4
  - files_markdown: 2.0.4
  - files_pdfviewer: 1.3.2
  - files_sharing: 1.6.2
  - files_texteditor: 2.6.0
  - files_trashbin: 1.4.1
  - files_versions: 1.7.1
  - files_videoplayer: 1.3.0
  - firstrunwizard: 2.3.0
  - fulltextsearch: 0.99.3
  - fulltextsearch_elasticsearch: 0.99.2
  - gallery: 18.1.0
  - jsloader: 1.0.1
  - logreader: 2.0.0
  - lookup_server_connector: 1.2.0
  - mail: 0.10.0
  - nextcloud_announcements: 1.3.0
  - notes: 2.4.2
  - notifications: 2.2.1
  - oauth2: 1.2.1
  - onlyoffice: 2.0.4
  - password_policy: 1.4.0
  - previewgenerator: 1.1.0
  - provisioning_api: 1.4.0
  - quota_warning: 1.3.0
  - serverinfo: 1.4.0
  - sharebymail: 1.4.0
  - socialsharing_email: 1.0.4
  - support: 1.0.0
  - survey_client: 1.2.0
  - systemtags: 1.4.0
  - tasks: 0.9.7
  - theming: 1.5.0
  - twofactor_backupcodes: 1.3.1
  - twofactor_totp: 1.5.0
  - updatenotification: 1.4.1
  - workflowengine: 1.4.0
Disabled:
  - admin_audit
  - circles
  - drawio
  - encryption
  - files_external
  - files_frommail
  - files_fulltextsearch_tesseract
  - socialsharing_facebook
  - socialsharing_googleplus
  - socialsharing_twitter
  - user_external
  - user_ldap
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "overwriteprotocol": "https",
        "trusted_domains": [
               "***REMOVED SENSITIVE VALUE***",
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "overwrite.cli.url": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "14.0.0.19",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "memcache.local": "\\OC\\Memcache\\APCu",
        "filelocking.enabled": "true",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": 6379,
            "timeout": 0
        },
        "mail_smtpmode": "smtp",
        "mail_smtpauthtype": "LOGIN",
        "maintenance": false,
        "htaccess.RewriteBase": "\/",
        "loglevel": 3,
        "theme": "",
        "mail_smtpauth": 1,
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "465",
        "mail_smtpsecure": "ssl",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "updater.release.channel": "stable",
        "integrity.check.disabled": false,
        "onlyoffice": {
            "verify_peer_off": true
        },
        "apps_paths": [
            {
                "path": "\/var\/www\/html\/apps",
                "url": "\/apps",
                "writable": false
            },
            {
                "path": "\/var\/www\/html\/custom_apps",
                "url": "\/custom_apps",
                "writable": true
            }
        ]
    }
}
```
</details>

**Are you using external storage, if yes which one:** none

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** none

### Client configuration
**Browser:** Chrome 69.0.3497.92 (Official Build) (64-bit)

**Operating system:** MacOS, Windows, Android, ... (happens everywhere

#### Web server error log
<details>
<summary>Web server error log</summary>

```
No error logs available for that specific issue
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>

```
No NC error logs available for that specific issue
```
</details>

#### Browser log
<details>
<summary>Browser log</summary>

```
No local errors
```
</details>

**Comment by Quinity** [comment]
![z1crnzx37d](https://user-images.githubusercontent.com/4330842/45619904-4370a680-ba7b-11e8-993b-04936a7215c3.gif)

**Comment by MorrisJobke** [comment]
@skjnldsv @nickvergessen Could this be a duplicate of #10934?

**Comment by nickvergessen** [comment]
I guess so.

**Comment by MorrisJobke** [comment]
Duplicate of #10934

**Comment by MorrisJobke** [comment]
This is not fixed by that fix #12231 -> so reopening.

@nickvergessen Any idea?

![bildschirmfoto 2018-11-04 um 14 28 29](https://user-images.githubusercontent.com/245432/47993263-543eaf80-e0ef-11e8-8341-f46e71aeed36.png)

**Comment by MorrisJobke** [comment]
> This is not fixed by that fix #12231 -> so reopening.

Fixed by #12577
